alerts/openshift-container-storage-operator/StorageAutoScalerCRIsInvalid.md

```diff
@@ -0,0 +1,27 @@
+# StorageAutoScalerCRIsInvalid
```

Comment:
@ShravaniVangur we can have different paramaters because of which invalid cr is detected,
```


    Duplicate cr

    lean mode detected

    Lso storageclass detected

```

so probably we need to document each one of them

Reply:
probably check that https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storageautoscaler/storageautoscaler_controller.go#L304